Generate.py

```
@@ -133,7 +134,12 @@ def main(args=None) -> tuple[argparse.Namespace, int]:
                     if yaml is None:
                         logging.warning(f"Ignoring empty yaml document #{doc_idx + 1} in {fname}")
                     else:
-                        weights_for_file.append(yaml)
+                        quantity = yaml.get("quantity", 1)
+                        if not allow_quantity and quantity > 1:
+                            raise ValueError("Quantity greater than 1 is deactivated by host settings.")
```

Sorry to be a bit annoying, but this `> 1` has made me realize I would like `<= 0` numbers to crash as well, rather than silently resulting in nothing :D

---

Negative numbers should fail just with `allow_quantity` set to `true` or everytime?